### ISSUE TITLE
Bug Report: `Assets.getPath()` no longer working as of v0.5.0

### ISSUE THREAD

**Issue by AbnormalPoof** [FunkinCrew/Funkin#3310]
### Issue Checklist

- [X] I have properly named the issue
- [X] I have checked the issues/discussions pages to see if the issue has been previously reported

### What platform are you using?

Itch.io (Downloadable Build) - MacOS

### If you are playing on a browser, which one are you using?

None

### Version

0.5.0

### Context (Provide images, videos, etc.)

OpenFL has a function in `openfl.utils.Assets` called `getPath()`. `getPath()` retrieves the full path of a specified file, it's really helpful for edge cases where you need the full file path.

I use this function for error handling in one of my mods, however, it seems to no longer work as of the latest update. (Code below for reference)
![Screenshot 2024-09-15 at 2 18 24 PM](https://github.com/user-attachments/assets/7d40e548-a7ae-4d33-936d-476c592c0128)
(This worked before on v0.4.1.)

I then created a test module, this module simply retrieves the full file path for Boyfriend's sprite sheet and outputs an alert.
```haxe
import funkin.modding.module.Module;
import openfl.utils.Assets;
import funkin.Paths;
import funkin.modding.PolymodErrorHandler;

class testModule extends Module {
  function new() {
    super('openflTestThing');
    var test = Assets.getPath(Paths.image('characters/BOYFRIEND'));
    if (test != null) PolymodErrorHandler.showAlert('TEST', 'IT WORKS');
  }
}
```

This code functions fine on v0.4.1.
![Screenshot 2024-09-15 at 2 15 04 PM](https://github.com/user-attachments/assets/9e200275-7402-4751-9f95-c74b6d8985cf)

v0.5.0, however...
![Screenshot 2024-09-15 at 2 15 26 PM](https://github.com/user-attachments/assets/a8280d0b-cb98-4d55-b228-86b233a7b0ac)
It seems as if the game can't find `getPath()` anymore, which is odd since it was working before on previous versions!

### Steps to reproduce (or crash logs, errors, etc.)

- Use any mod that uses `Assets.getPath()`
- Script error

**Comment by AbnormalPoof** [comment]
Just looked at the JS code, and yeah...
![Screenshot 2024-09-15 at 2 27 44 PM](https://github.com/user-attachments/assets/ffc2a3ff-7318-4d4a-a12d-e80fb0a6884f)
Seems as if whatever this class is doesn't seem to have `getPath()` as a function.

**Comment by AbnormalPoof** [comment]
Fixed in v0.5.1
